upload_parrots.py

```diff
@@ -70,6 +90,8 @@ def main():
                         help='The slack username with which to upload')
     parser.add_argument('--password',
                         help='The slack password with which to login')
+    parser.add_argument('--google',
```

Comment:
this could be made optional and default to `False` to make the experience smoother. Here's what it would look like if it used `store_true`:
https://docs.python.org/2/library/argparse.html#action